### PR TITLE
Adaptor to springboot2.1.9

### LINKED ISSUE #13: 适配 spring-boot2.1.9-release 等相关组件

### PR DESCRIPTION
**Pull request by gaosaroma**
### Motivation

适配 springboot 2.1.9

Adapt to springboot 2.1.9

### Modification

- 降低 oshi 版本至 3.9.1
- 根据springboot2.1.9中log4j2的实现，添加适配器 koupleless-adapter-log4j2-springboot2.1.9
- 适配springboot2.1.9版本的 AbstractEnvironment

- Downgrade oshi version to 3.9.1.
- Add an adapter koupleless-adapter-log4j2-springboot2.1.9 based on the implementation of log4j2 in springboot 2.1.9.
- Adapt AbstractEnvironment to springboot 2.1.9 version.

### Verification

- 验证 tomcat sample
- 验证 log4j2 sample

- Validate in tomcat sample
- Validate in log4j2 sample

### How to use
- 在基座启动类上，排除 com.alipay.sofa.koupleless.arklet.springboot.starter.health.HealthAutoConfiguration
- Exclude "com.alipay.sofa.koupleless.arklet.springboot.starter.health.HealthAutoConfiguration" in the BaseApplication class.
image

- 在基座pom中，排除原适配器 koupleless-adapter-log4j2，使用适配器 koupleless-adapter-log4j2-springboot2.1.9
- Exclude the original adapter "koupleless-adapter-log4j2" and use the adapter "koupleless-adapter-log4j2-springboot2.1.9" in the baseApplication pom.
image

### Result

Resolved or fixed #13

### CODE REVIEW

**Comment by gaosaroma** [comment]
注意：未处理"HealthAutoConfiguraion"的适配问题，须后续重构 HealthAutoConfiguraion。

**Comment by lvjing2** [comment]
@thinkergithub

File: koupleless-runtime/koupleless-adapter-ext/koupleless-adapter-log4j2-springboot2.1.9/.gitignore
Comment (lvjing2) [inline koupleless-runtime/koupleless-adapter-ext/koupleless-adapter-log4j2-springboot2.1.9/.gitignore:1]: 这个不用提交

**Review by lvjing2 — APPROVED** [review]
LGTM

**Pull request merged**